modules/component/src/let/let.directive.ts

```diff
@@ -15,78 +15,72 @@ import { createRenderScheduler } from '../core/render-scheduler';
 import { createRenderEventManager } from '../core/render-event/manager';
 
 export interface LetViewContext<T> {
-  // to enable `let` syntax we have to use $implicit (var; let v = var)
+  /**
+   * using `$implicit` to enable `let` syntax: `*ngrxLet="obs$; let o"`
+   */
   $implicit: T;
-  // to enable `as` syntax we have to assign the directives selector (var as v)
+  /**
+   * using `ngrxLet` to enable `as` syntax: `*ngrxLet="obs$ as o"`
+   */
   ngrxLet: T;
-  // set context var complete to true (var$; let e = $error)
+  /**
+   * `*ngrxLet="obs$; let e = $error"` or `*ngrxLet="obs$; $error as e"`
+   */
   $error: boolean;
-  // set context var complete to true (var$; let c = $complete)
+  /**
+   * `*ngrxLet="obs$; let c = $complete"` or `*ngrxLet="obs$; $complete as c"`
+   */
   $complete: boolean;
+  /**
+   * `*ngrxLet="obs$; let s = $suspense"` or `*ngrxLet="obs$; $suspense as s"`
+   */
+  $suspense: boolean;
 }
 
 /**
  * @ngModule ReactiveComponentModule
  *
  * @description
  *
- * The `*ngrxLet` directive serves a convenient way of binding observables to a view context (a dom element scope).
- * It also helps with several internal processing under the hood.
- *
- * The current way of binding an observable to the view looks like that:
- * ```html
- * <ng-container *ngIf="observableNumber$ | async as n">
- * <app-number [number]="n">
- * </app-number>
- * <app-number-special [number]="n">
- * </app-number-special>
- * </ng-container>
- *  ```
- *
- *  The problem is `*ngIf` is also interfering with rendering and in case of a `0` the component would be hidden
- *
- * Included Features:
- * - binding is always present. (`*ngIf="truthy$ | async"`)
- * - it takes away the multiple usages of the `async` or `ngrxPush` pipe
- * - a unified/structured way of handling null and undefined
- * - triggers change-detection differently if `zone.js` is present or not (`ChangeDetectorRef.detectChanges` or `ChangeDetectorRef.markForCheck`)
- * - triggers change-detection differently if ViewEngine or Ivy is present (`ChangeDetectorRef.detectChanges` or `ɵdetectChanges`)
- * - distinct same values in a row (distinctUntilChanged operator)
```

Comment:
Removed duplicated content that is also in the `let.md` file. I have left a short description and examples here.